src/main/java/com/crudtest/controller/UserController.java

```diff
@@ -34,10 +37,6 @@ public ModelAndView registration(){
     @RequestMapping(value="/", method = RequestMethod.POST)
     public ModelAndView addUser(@Valid User user, BindingResult bindingResult){
```

Comment:
Why are you using @Valid annotation when there is no annotation on your User POJO or a separate Validator class that handles it.